lms/djangoapps/commerce/api/v1/views.py

```diff
@@ -64,10 +73,17 @@ class OrderView(APIView):
     """ Retrieve order details. """
 
     authentication_classes = (SessionAuthentication,)
-    permission_classes = (IsAuthenticated,)
+    permission_classes = (IsAuthenticatedOrActivationOverridden,)
```

Comment:
So unless you feel strongly about this particular bit, I'm going to leave it in -- the page technically still exists so it should work properly.  If we want to take the entire page out at some point that is fine, but for now I'd like to fix it.